src/Microsoft.DotNet.Darc/Darc/Operations/GetAssetOperation.cs

```diff
@@ -60,6 +66,7 @@ public override async Task<int> ExecuteAsync()
                 (await _barClient.GetAssetsAsync(name: _options.Name, version: _options.Version, buildId: _options.Build)).ToList();
 
             var queryDescription = new StringBuilder();
+            queryDescription.Append(_options.Latest ? " latest asset" : " assets");
```

Comment:
why put the whitespace here and not in the `Console.WriteLine`?

Reply:
Continuity. All the builder appends have a starting space, so I just kept it like that